api/tests/unit/domain/validations/user-validator_test.js

```diff
@@ -181,6 +181,26 @@ describe('Unit | Domain | Validators | user-validator', function() {
           }
         });
 
+        it('should reject with error when neither email or username are defined', () => {
```

Comment:
```suggestion
        it('should reject with error when neither email **nor** username are defined', () => {
```
[Emmy, es-tu là ?](https://www.dailywritingtips.com/understanding-either-or-and-neither-nor/)

Reply:
Done :)